src/@context/UserPreferences.tsx

```diff
@@ -40,97 +45,161 @@ interface UserPreferencesValue {
 
 const UserPreferencesContext = createContext(null)
 
-const localStorageKey = 'ocean-user-preferences-v4'
-
-function getLocalStorage(): UserPreferencesValue {
-  const storageParsed =
-    isBrowser && JSON.parse(window.localStorage.getItem(localStorageKey))
-  return storageParsed
-}
-
-function setLocalStorage(values: Partial<UserPreferencesValue>) {
-  return (
-    isBrowser &&
-    window.localStorage.setItem(localStorageKey, JSON.stringify(values))
-  )
-}
-
 function UserPreferencesProvider({
   children
 }: {
   children: ReactNode
 }): ReactElement {
   const { appConfig } = useMarketMetadata()
-  const localStorage = getLocalStorage()
   // Set default values from localStorage
-  const [debug, setDebug] = useState<boolean>(localStorage?.debug || false)
+  const [debug, setDebug] = useState<boolean>(
+    getCookieValue('debug') === 'true'
+  )
   const [currency, setCurrency] = useState<string>(
     localStorage?.currency || 'EUR'
   )
   const [locale, setLocale] = useState<string>()
-  const [bookmarks, setBookmarks] = useState(localStorage?.bookmarks || [])
+  const [bookmarks, setBookmarks] = useState(
+    getCookieValue('bookmarks')?.split(',') || []
+  )
   const [chainIds, setChainIds] = useState(
-    localStorage?.chainIds || appConfig.chainIds
+    getCookieValue('chainIds')?.split(',').map(Number) || appConfig.chainIds
   )
   const { defaultPrivacyPolicySlug, showOnboardingModuleByDefault } = appConfig
 
   const [privacyPolicySlug, setPrivacyPolicySlug] = useState<string>(
-    localStorage?.privacyPolicySlug || defaultPrivacyPolicySlug
+    getCookieValue('privacyPolicySlug') || defaultPrivacyPolicySlug
   )
 
   const [showPPC, setShowPPC] = useState<boolean>(
-    localStorage?.showPPC !== false
+    getCookieValue('showPPC') !== 'false'
   )
 
   const [allowExternalContent, setAllowExternalContent] = useState<boolean>(
-    localStorage?.allowExternalContent || false
+    getCookieValue('allowExternalContent') === 'true'
   )
 
   const [automationWallet, setAutomationWallet] = useState<string>(
-    localStorage?.automationWalletJSON || ''
+    typeof getCookieValue('automationWallet') !== 'undefined'
+      ? JSON.parse(getCookieValue('automationWallet'))?.automationWalletJSON
+      : ''
   )
 
   const [automationWalletMode, setAutomationWalletMode] =
     useState<AUTOMATION_MODES>(
-      localStorage?.automationWalletMode || AUTOMATION_MODES.SIMPLE
+      typeof getCookieValue('automationWallet') !== 'undefined'
+        ? JSON.parse(getCookieValue('automationWallet'))?.automationWalletMode
+        : AUTOMATION_MODES.SIMPLE
```

Comment:
suggestion(non-blocking): would be nice if this can be configured in app.config per portal